portfolio/src/main/webapp/script.js

```diff
@@ -25,7 +25,8 @@ const idOfContactList = "contact-list";
 let positions = [ 'Freelance Web Developer', 'Robotics Builder', 'Graphic Designer', 'Entrepreneur' ];
 let i = 0;
 let contactIds = [];
-
+let onlyElementInArray = 0; 
+let checkIfAnElementIsEmpty = 0;
```

Comment:
There isn't anything explicitly wrong here, but these variable names don't quite align with their value. Specifically, names that begin with verbs are typically reserved for methods. Also, these seem to have the same use case, and are set to the same value. Do you need both?

Reply:
the thing, is the index of the first element in an array is 0 and the length of an empty array is 0, so technically I thought I would need both of these variables.

Reply:
This is correct, but try to think within the context of the application. These are two well know attributes of how arrays work, that don't need to be re-defined here, but perhaps they have special implications for this application?

Reply:
yes, because one variable gets the first element in the array and the checkIfAnElmentIsempty checks if the array is empty and thus waiting for an image to be uploaded